README.md

```diff
@@ -26,5 +26,61 @@ I recommend using the open source [sqlitebrowser](http://sqlitebrowser.org/) to
 ![sqllitebrowser_screenshot](
 https://github.com/sqlitebrowser/sqlitebrowser/raw/master/images/sqlitebrowser.png)
 
+You can also upload the created `*.sql3` file to https://sqliteonline.com/, saving yourself from needing to download any applications.
+
+## Usage Examples
+
+### SQL
+
+- `SELECT * FROM  nutrient;` - print out all nutrient data
```

Comment:
I like these examples. Could you format them consistently? E.g. single space between words and a space after comma.
Thanks!